### PR TITLE
[v2.9] Update rancher-vsphere-csi

### PR DESCRIPTION
**Pull request by HarrisonWAffel**
## Problem

The vSphere CSI chart has been updated to include a new field in the config map, https://github.com/rancher/vsphere-charts/pull/75

## Solution

Update the upstream commit for `rancher-vsphere-csi` in order to pull in the updated config map for the 2.9 release line

### CODE REVIEW

**Review by jiaqiluo — APPROVED** [review]

File: index.yaml [hunk 1]
```diff
@@ -6084,30 +6084,6 @@ entries:
     urls:
     - assets/prometheus-federator/prometheus-federator-104.0.0-rc1+up0.4.1.tgz
     version: 104.0.0-rc1+up0.4.1
-  - annotations:
```
Comment (adamkpickering) [inline index.yaml]: Did you mean to delete this block? I don't see how this relates to this PR.

**Review by adamkpickering — CHANGES_REQUESTED** [review]

File: index.yaml (hunk 1 above)
Comment (HarrisonWAffel) [reply index.yaml]: Hmm, no I didn't. It looks like this was done automatically when using the charts build scripts. Let me see if I can avoid this from happening

Comment (HarrisonWAffel) [reply index.yaml]: Okay, most recent pushes have resolved this issue

**Review by adamkpickering — APPROVED** [review]

File: index.yaml (hunk 1 above)
Comment (jiaqiluo) [reply index.yaml]: @adamkpickering  good catch!

**Pull request merged**